scripts/x86.sh

```diff
@@ -63,6 +63,10 @@ if [[ -v TOUCH_CALIBRATION ]]; then
   xinput set-int-prop "eGalax Inc. USB TouchController Touchscreen" "Evdev Axis Calibration" $TOUCH_CALIBRATION
 fi
 
+if [[ -v TRANSFORM_CALIBRATION ]]; then
+  xinput set-prop "eGalax Inc. USB TouchController Touchscreen" "Coordinate Transformation Matrix" $TRANSFORM_CALIBRATION
```

Comment:
```suggestion
  xinput set-prop "eGalax Inc. USB TouchController Touchscreen" --type=float "Coordinate Transformation Matrix" $TRANSFORM_CALIBRATION
```

Do we also need to set the type to float? I presume we're going to be transforming the matrix by small values?